codec-http2/src/main/java/io/netty/handler/codec/http2/Http2MultiplexHandler.java

```diff
@@ -277,6 +279,17 @@ public void exceptionCaught(ChannelHandlerContext ctx, Throwable cause) throws E
             }
             return;
         }
+        if (cause.getCause() instanceof SSLException) {
```

Comment:
I wonder if this condition is overly narrow. Like, are we sure we only need to do this for SSLExceptions; and only ones that directly cause the current cause; it can't be higher or lower in the chain, etc…

Reply:
It can't be higher or lower in the chain... I did go with SSLException only as this is the only case I am aware of atm.